src/bundle/Resources/public/js/scripts/core/dropdown.js

```diff
@@ -444,16 +444,14 @@
                 return;
             }
 
-            const modalDialog = this.container.closest('.modal-dialog');
-
             this.itemsPopover = new DropdownPopover(
                 this.selectedItemsContainer,
                 {
                     html: true,
                     placement: 'bottom',
                     customClass: 'ibexa-dropdown-popover',
                     content: this.itemsPopoverContent,
-                    container: modalDialog || 'body',
```

Comment:
Having a dropdown popup inside a modal caused problems when the modal was embedded inside a tab.